test/e2e/logforwarding/fluent/fluent_test.go

```diff
@@ -15,18 +16,22 @@ import (
 const message = "My life is my message"
 
 var _ = Describe("[ClusterLogForwarder]", func() {
+	const basePort = 24224
 	var (
-		c *client.Test
-		f *Fixture
+		c           *client.Test
+		f           *Fixture
+		testOrdinal int
+		logTypes    = []string{"application", "infrastructure", "audit"}
 	)
 	BeforeEach(func() { c = client.NewTest(); f = NewFixture(c.NS.Name, message) })
 	AfterEach(func() { c.Close() })
 
 	Context("with app/infra/audit receiver", func() {
 		BeforeEach(func() {
-			f.Receiver.AddSource(&fluentd.Source{Name: "application", Type: "forward", Port: 24224})
-			f.Receiver.AddSource(&fluentd.Source{Name: "infrastructure", Type: "forward", Port: 24225})
-			f.Receiver.AddSource(&fluentd.Source{Name: "audit", Type: "forward", Port: 24226})
+			for i, logType := range logTypes {
+				f.Receiver.AddSource(&fluentd.Source{Name: logType, Type: "forward", Port: basePort + len(logTypes)*testOrdinal + i})
```

Comment:
can simplify this by `Port: basePort + testOrdinal` and incrementing `testOrdinal` in the loop